### PR TITLE
CLDR-13928 Value for 20-vote item moved to Winning column with 8 votes

### PR DESCRIPTION
**Pull request by btangmu**
-Add tests for two rules:

--- (1) For a 20 vote item, 8 votes are needed to change from non-approved to approved

--- (2) Once Approved, it requires 20 votes to change

-Revise VoteResolver to pass both tests; adjust from HIGH_BAR (20) to LOWER_BAR (8) if not approved

-Revise SurveyJSONWrapper.wrap(VoteResolver) to send client the adjusted required votes

-Remove unneeded 3rd arg of computeStatus (always trunkStatus)

CLDR-13928

- [x] This PR completes the ticket.

### CODE REVIEW

**Review by macchiati — APPROVED** [review]

**Comment by btangmu** [comment]
I'll wait until after today's push to production to merge this.

**Comment by macchiati** [comment]
sounds good. This would revert any winning values anyway after we deploy,
so there is no big urgency.
Mark

File: tools/cldr-apps/src/test/resources/org/unicode/cldr/unittest/web/data/TestSTFactory.xml
Comment (srl295) [inline tools/cldr-apps/src/test/resources/org/unicode/cldr/unittest/web/data/TestSTFactory.xml:213]: it might be good to have such a mock.

We already have the sandbox locales ( see CLDR-12002 ) so there's precedent for having multiple input dirs.  Actually, a `mul` sublocale such as `mul-AA` might make sense. it could have static test data, and get copied into the sandbox directory on startup for test use.

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java:744]:
```suggestion
     * What are the adjusted required votes for this item, taking into account the trunkStatus?
```

Comment (srl295) [reply tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java:744]: I would also update getRequiredVotes to say that it _only_ takes the xpath and locale into account. 

`canFlagOnLosing()` wasn't updated, should it be? 

Perhaps this function should be `getRequiredVotes()` as this is the _actual_ required votes, and the other one renamed to `getUnadjustedRequiredVotes()`

I'm splitting hairs here, but this is critical core machinery, and can be challenging to trace through. ('Adjusted' doesn't tell me much about what is happening, and I know we have several 'adjusted' functions in various parts now)

**Review by srl295 — APPROVED** [review]
LGTM with usual rants about naming, this is software after all :)

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java
Comment (btangmu) [reply tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java:744]: canFlagOnLosing works correctly, as it stands, since requiredVotes == HIGH_BAR even when getAdjustedRequiredVotes may return LOWER_BAR

I like your suggestions -- the "adjusted" business has a disagreeable smell! there's definitely a lot of room for improvement in the implementation and naming and comments. I was trying to avoid touching/changing more of the code for this PR than was really necessary to accomplish the goal of this ticket; maybe too cautious...

**Comment by btangmu** [comment]
I forgot to merge this a week ago! The plan was to wait until after a push to production that was about to happen. I've checked there aren't more recent changes to the 3 files, so I'll merge now

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java
Comment (btangmu) [reply tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java:744]: actually maybe I was wrong about canFlagOnLosing -- I'll email cldr-core2 for policy clarification

**Pull request merged**
